src/common.rs

```diff
@@ -83,6 +83,9 @@ pub(crate) fn fix_symbol_name<'a>(name: &'a Name<'a>) -> Name<'a> {
     }
     let fixed = LLVM_NNN.replace(name.as_str(), "");
 
+    // Normalize anonymous namespace symbol between platforms
```

Comment:
I noticed this file has no comments. I'm game for removing this one if it's unhelpful.

Reply:
Turn it into a small function with a descriptive name instead, like `normalize_anonymous_namespace()` that's the best way to document code w/o risking for the comments to go out-of-date. I'm always super positive about splitting out one- or two-line functions if it means describing what's happening; it makes reading the code a smoother experience.